Cargo.toml

```diff
@@ -133,7 +136,7 @@ thiserror = "1"
 time = { version = "0.3", features = ["serde"] }
 tokio = { version = "1.20", features = ["fs"] }
 tracing = { version = "0.1", optional = true }
-ureq = { version = "2", features = ["rustls-native-certs"] }
+ureq = { version = "2", default-features = false, features = ["gzip"] }
```

Comment:
We don't need `gzip` too.

Reply:
Maybe it's better to leave it here for better performance? It's one of `ureq`'s default features to support `Content-Encoding: gzip` I think.

Reply:
AFAIK, We will not use gzip features provided by `ureq` or `reqwest` because we are talking to storage services like `s3`, `gcs`.